types/stampit/stampit-tests.ts

```diff
@@ -149,15 +173,126 @@ interface SomeStampInstance {
 }
 
 // Test import of stamp type
-interface SomeStamp extends stampit.Stamp {
+interface SomeStamp extends stampit.Stamp<SomeStampInstance> {
     (params: { a: number; b: boolean}): SomeStampInstance;
 }
 
-const SomeStamp = stampit()
+const SomeStamp = stampit<SomeStamp>()
     .init(function(params: { a: number; b: boolean}) {
         this.a = '' + a;
         this.b = '' + b;
-    }) as SomeStamp;
+    });
 
 SomeStamp({ a: 1, b: false }); // $ExpectType SomeStampInstance
 SomeStamp({ a: 1, b: false }).a; // $ExpectType string
+
+const d: stampit.Descriptor<{}> = {
```

Comment:
This should be `stampit.ExtendedDescriptor`

Reply:
In the context of `stampit` types, what would be the benefit of distinguishing Stamp's `Descriptor` from `stampit`'s `Descriptor`?